diff -up attr-2.4.48/test/run.omv~ attr-2.4.48/test/run
--- attr-2.4.48/test/run.omv~	2018-09-04 04:50:02.091320164 +0200
+++ attr-2.4.48/test/run	2018-09-04 04:50:11.046190685 +0200
@@ -106,7 +106,7 @@ for (;;) {
   if (defined $line) {
     # Substitute %VAR and %{VAR} with environment variables.
     $line =~ s[%(\w+)][$ENV{$1}]eg;
-    $line =~ s[%{(\w+)}][$ENV{$1}]eg;
+    $line =~ s[%\{(\w+)\}][$ENV{$1}]eg;
   }
   if (defined $line) {
     if ($line =~ s/^\s*< ?//) {
